public/app/features/variables/editor/VariableEditorEditor.tsx

```diff
@@ -204,7 +204,11 @@ export class VariableEditorEditorUnConnected extends PureComponent<Props, State>
                 Run query
                 {loading && <Icon className="spin-clockwise" name="sync" size="sm" style={{ marginLeft: '2px' }} />}
               </Button>
-              <Button variant="primary" onClick={this.onApply}>
+              <Button
+                variant="primary"
+                onClick={this.onApply}
+                aria-label={selectors.pages.Dashboard.Settings.Variables.Edit.General.applyButton}
```

Comment:
rather than use an aria label maybe we can use a data test id :) since I think this button should already be well labelled by the text that lives in side it, yeah? https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-label

Reply:
Yep I can take a look at moving to `data-testid`. I think that is meant to be the standard anyway 🤔